shuup/campaigns/migrations/0013_hourconfition_fix.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-06-11 22:27
+from __future__ import unicode_literals
+
+import django.core.validators
+import django.utils.version
+from django.db import migrations, models
+import re
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('campaigns', '0012_basket_campaign_undiscounted'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='hourbasketcondition',
+            name='days',
+            field=models.CharField(verbose_name='days', max_length=255, validators=[django.core.validators.RegexValidator(re.compile('^[\\d,]+\\Z' if django.utils.version.get_docs_version() == "1.8" else '^\\d+(?:\\,\\d+)*\\Z', 32), code='invalid', message='Enter only digits separated by commas.')]),
```

Comment:
You changed this migration manually, did you?

How about wrapping the code so that it can be read more easily? :)

Also, rather than using the `get_docs_version` just `import django`  and do `if django.VERSION < (1, 10)`.